.eslintrc.js

```diff
@@ -5,6 +5,10 @@ module.exports = {
       files: ['**/*.test.{cjs,js,mjs}'],
       env: {
         jest: true
+      },
+      globals: {
+        page: true,
+        browser: true
```

Comment:
Allows the `jest-puppeteer` **page** and **browser** globals to work without lint errors